plugins/main/public/app.js

```diff
@@ -105,23 +105,29 @@ app.run(['$rootElement', function ($rootElement) {
   addHelpMenuToAppChrome();
 
   // Bind deleteExistentToken on Log out component.
-  $('.euiHeaderSectionItem__button, .euiHeaderSectionItemButton').on('mouseleave', function () {
-    // opendistro
-    $('button:contains(Log out)').on('click', function () {
-      WzAuthentication.deleteExistentToken();
-    });
-    // x-pack
-    $('a:contains(Log out)').on('click', function (event) {
-      // Override href's behaviour and navigate programatically
-      // to the logout path once the token has been deleted.
-      event.preventDefault();
-      WzAuthentication.deleteExistentToken()
-        .catch((err) => {
-          console.error('[ERROR] - User token could not be deprecated - ', err);
-        })
-        .finally(() => {
-          window.location = event.currentTarget.href;
-        });
-    });
-  });
-}]);
+  $('.euiHeaderSectionItem__button, .euiHeaderSectionItemButton').on(
+    'mouseleave',
+    function () {
+      // opendistro
+      $('button:contains(Log out)').on('click', function () {
+        WzAuthentication.deleteExistentToken();
+      });
+      // x-pack
+      $('a:contains(Log out)').on('click', function (event) {
+        // Override href's behaviour and navigate programatically
+        // to the logout path once the token has been deleted.
+        event.preventDefault();
+        WzAuthentication.deleteExistentToken()
+          .catch(err => {
+            console.error(
+              '[ERROR] - User token could not be deprecated - ',
+              err,
+            );
+          })
+          .finally(() => {
+            window.location = event.currentTarget.href;
+          });
+      });
+    },
+  );
+});
```

Comment:
Something is wrong at the end of the function, it's missing a bracket. Please check there wasn't an error in the merge.

Reply:
Good catch. I did not see that the closing bracket was missing due to code formatting. Good catch. Fixed in 6e9e4b7cbddd81b316e3ff004262516653efdf82.